vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,379 @@
+
+# GSEA using linear models
+
+GSEA is a popular method that can be used to assess whether or not particular gene sets are associated with a phenotype of interest [@Subramanian2005; @Tian2005; @JiangGent2007]. Most applications of this method do not explicitly deal with structure of the gene sets, but when analyzing chromosomal location such methods are desirable. We present a simple approach that is similar in spirit to traditional GSEA, and generalizes nicely to accommodate nested categories. Consider the situation where gene $i$ has an associated measure of differential expression $y_i$; for example, an attenuated $t$-statistic derived from a differential expression analysis such as [limma]{.sans-serif} [@limma]. Given a particular category, GSEA asks whether the distribution of $y_i$-s restricted to the category of interest is "unusual". Thus, in<span><a href="#fig:gseaStripplot"> Figure 2</a><span>, we might be interested in knowing whether the distribution of $y_i$ values for genes in 12q21 is different from that of the genes in 12q (for a local test) or of all genes in the gene universe (for a global test).<a href="#fig:gseaStripplot"> Figure 2</a> is produced by
+
+<figure id="fig:gseaStripplot">
+```{r fig=TRUE, fig.align="center",width=9, height=4}
+gseaTstatStripplot(c("12q21.1", "12q21", "12q2", "12q"),
+                   include.all = TRUE, 
+                   g = g1,
+                   xlab = "Per-gene t-statistics", 
+                   panel = function(...) {
+                       require(grid, quietly = TRUE)
+                       grid.rect(y = unit(2, "native"), 
+                                 height = unit(1, "native"),
+                                 gp = 
+                                 gpar(fill = "lightgrey", 
+                                      col = "transparent"))
+                       panel.grid(v = -1, h = 0)
+                       panel.stripplot(...)
+                       panel.average(..., fun = mean, lwd = 3)
+                   })
+```
+
+```{r gseaStripplot, fig=TRUE, fig.align="center",include=FALSE, prefix=FALSE, echo=FALSE, prefix=FALSE, width=9, height=4}
+plot(trellis.last.object())
+```
+
+<figcaption>Figure 2: Per-gene [<em>t</em>]{.math .inline}-statistics, as computed by [limma]{.sans-serif}, for selected chromosome bands. The points are jittered vertically to alleviate overlap. A thick grey line joins the mean value within each group. A GSEA test would compare the genes in 12q21 with those in 12q, or the entire gene universe, by fitting a linear model with per-gene [<em>t</em>]{.math .inline}-statistics as the response, and a term indicating membership in 12q21 as the predictor. If 12q21.1 is declared as significant, a conditional GSEA test would include an additional term for 12q21.1.</figcaption>
```

Comment:
- [ ] Make link for limma
- [ ] Replace `[<em>t</em>]{.math .inline}` with `[<em>t</em>]{.math .inline}-statistics` with `$t$-statistics`